docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -109,10 +109,31 @@ besu operator generate-blockchain-config --config-file=<FILE> --to=<DIRECTORY> [
 besu operator generate-blockchain-config --config-file=config.json --to=myNetworkFiles
 ```
 
-The configuration file has 2 subnested JSON nodes. The first is the `genesis` property defining 
-the [IBFT 2.0 genesis file](../../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file) except for the `extraData` string. The 
+The configuration file has 2 subnested JSON nodes. The first is the `genesis` property defining
+the [IBFT 2.0 genesis file](../../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file) except for the `extraData` string. The
 second is the `blockchain` property defining the number of key pairs to generate.  
 
+### generate-log-bloom-cache
+
+Generates cached log bloom indexes for blocks. APIs use the cached indexes for improved log
```

Comment:
Do we want to note that this cannot be done against a running node or is that implicit?

Reply:
@MadelineMurray, do we expect users to know this? For example we don't call this out for `public-key export-address`, but maybe we should?